### ISSUE TITLE
changing mode to Static or Kinematic of RigidBody causes runtime error

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#15743]
**Godot version:**

Godot 3.0 RC1 05b184381

**OS/device including version:**

Ubuntu 17.10

**Issue description:**

changing mode to Static or Kinematic of RigidBody causes runtime error with bullet physics
```
0:00:00:0183 - Condition ' PhysicsServer::BODY_MODE_RIGID != mode && PhysicsServer::BODY_MODE_CHARACTER != mode ' is true.
----------
Type:Error
Description: 
Time: 0:00:00:0183
C Error: Condition ' PhysicsServer::BODY_MODE_RIGID != mode && PhysicsServer::BODY_MODE_CHARACTER != mode ' is true.
C Source: modules/bullet/rigid_body_bullet.cpp:958
C Function: _internal_set_mass
```

**Steps to reproduce:**
1. Ensure current physics 3d engine is `Default` or `Bullet`
2. Add `RigidBody` to scene
3. Change 'Mode' to `Static` or `Kinematic`
4. Run scene

**Minimal reproduction project:**

[Test Physics 1.zip](https://github.com/godotengine/godot/files/1631672/Test.Physics.1.zip)

**Comment by akien-mga** [comment]
CC @AndreaCatania

**Comment by AndreaCatania** [comment]
This is a fix for this Issue #15743
